### PR TITLE
Add ✕ button to end search

### LINKED ISSUE #2551: Add an X to allow to clear inputes
Applicable for:
* Search/filter conversations
* Adding participants in the sidebar
* Selecting participants in the create dialog

### PR DESCRIPTION
**Pull request by ma12-co**
Adds a button that allows to end the search operation.

Fix #2551

### CODE REVIEW

**Comment by ma12-co** [comment]
ping @nickvergessen

**Comment by nickvergessen** [comment]
`2. developing` but sure ;)

**Comment by nickvergessen** [comment]
Any update @ma12-co

**Comment by ma12-co** [comment]
Ready @nickvergessen

File: src/components/RightSidebar/Participants/ParticipantsTab.vue [hunk 1]
```diff
@@ -25,7 +25,9 @@
 			v-if="displaySearchBox"
 			v-model="searchText"
 			:placeholder-text="t('spreed', 'Add participants to the conversation')"
-			@input="handleInput" />
+			@input="handleInput"
+			@abort-search="abortSearch"
+			:is-searching="isSearching" />
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/ParticipantsTab.vue]: > Attribute ":is-searching" should go before "@abort-search"

File: src/components/RightSidebar/Participants/ParticipantsTab.vue [hunk 2]
```diff
@@ -346,6 +348,10 @@ export default {
 				}
 			}
 		},
+		// Ends the search operation
+		abortSearch() {
+			this.searchText = ''
+		}
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/ParticipantsTab.vue]: > Missing trailing comma

Comment (ma12-co) [reply src/components/RightSidebar/Participants/ParticipantsTab.vue]: Shouldn't I get these in the output of make watch-js?

**Pull request merged**